napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -40,18 +40,31 @@
 from ..widgets.qt_plugin_sorter import QtPluginSorter
 from .qt_plugin_report import QtPluginErrReporter
 
-# TODO: add error icon and handle pip install errors
-
 
+# TODO: add error icon and handle pip install errors
 # TODO: add queue to handle clicks when already processing
 class Installer:
     def __init__(self, output_widget: QTextEdit = None):
         from ...plugins import plugin_manager
 
+        # To be used when conda is fully supported
+        # self._use_conda = _is_installed_with_conda()
+        self._use_conda = False
+        self._conda_env_path = None
+
+        if self._use_conda and (Path(sys.prefix) / "conda-meta").is_dir():
+            self._conda_env_path = sys.prefix
```

Comment:
Rather than making this a private variable, can we make this a constructor argument? Similarly, `install` should take in a `install_method=` kwarg that can override the current setting. ie let's make things depend as little as possible on global state, even if global state is the fallback.

Reply:
Fixed!